omod/src/main/webapp/resources/htmlforms/oncologyEnrollment.html

```diff
@@ -31,7 +31,32 @@
     </div>
     <div class="ke-form-content">
         <fieldset>
+            <div class="ke-form-content">
 
+                <legend>Religion Source</legend>
+                <table>
+                    <tr>
+                        <td>Employment Status</td>
+
+                    </tr>
+                    <tr>
+                        <td>Health Insurance</td>
+
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><obs id="Smokes Cigarrets" conceptId="159424" answerConceptIds="1065,1066" answerLabels= "1-Yes,2-No" style="radio"/></td>
```

Comment:
Please apply the uuids as oppossed to concept ids

Reply:
This has been noted and changed